cli/dcoscli/main.py

```diff
@@ -140,3 +154,19 @@ def wrapper(*args, **kwargs):
             return 1
         return result
     return wrapper
+
+
+def set_ssl_info_env_vars(config):
+    """Set SSL info from config to environment variables
+
+    :param config: config
+    :type config: Toml
+    :rtype: None
+    """
+
+    if 'core.ssl_verify' in config:
+        os.environ[constants.DCOS_SSL_VERIFY_ENV] = str(
+            config['core.ssl_verify'])
```

Comment:
This means that `dcos.toml` takes precedence over the `DCOS_SSL_VERIFY` environment variable. Is this what we want? As a user this would be surprising to me.